src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinAnnotationIntrospector.kt

```diff
@@ -65,40 +65,34 @@ internal class KotlinAnnotationIntrospector(private val context: Module.SetupCon
     // Find a serializer to handle the case where the getter returns an unboxed value from the value class.
     override fun findSerializer(am: Annotated): StdSerializer<*>? = when (am) {
         is AnnotatedMethod -> {
-            when (KotlinVersion.CURRENT.isAtLeast(1, 5)) {
```

Comment:
This has been removed as it is no longer needed once support for `Kotlin 1.4` and below is removed.